numpy/random/mtrand/mtrand.pyx

```diff
@@ -1964,8 +1971,14 @@ cdef class RandomState:
             The scale parameter, :math:`\\beta = 1/\\lambda`.
         size : int or tuple of ints, optional
             Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
-            ``m * n * k`` samples are drawn.  Default is None, in which case a
-            single value is returned.
+            ``m * n * k`` samples are drawn.  If size is ``None`` (default),
+            a single value is returned if ``scale`` is a scalar.  Otherwise,
+            ``len(scale)`` samples are drawn.
```

Comment:
This is not exactly correct: if `scale` is a 2D array, `len(scale)` is its number of rows, and this function returns `scale.size` items. If we wanted to write it in code, `np.array(scale).size` is probably the most accurate representation, but it may be better to come up with a description in prose, not sure...